pom.xml

```diff
@@ -32,6 +33,9 @@
         <version.org.jboss.wildfly.dist>18.0.0.Final</version.org.jboss.wildfly.dist>
         <version.org.wildfly.arquillian>2.2.0.Final</version.org.wildfly.arquillian>
         <version.org.fusesource.jansi>1.18</version.org.fusesource.jansi>
+        <version.commons-io.commons-io>2.6</version.commons-io.commons-io>
```

Comment:
Still not sure we need all of them :) Try to convince me

Reply:
https://github.com/jboss-eap-qe/eap-microprofile-test-suite/issues/39#issuecomment-558021120